packages/presentation/lib/utils/extensions/extention_list.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:presentation/utils/extensions/extention_string.dart';
+
+extension ListExtention on List<String> {
+  String capitalizeListFirstOfEach() {
+    return join(", ").split(" ").map((str) => str.capitalize).join(", ");
+  }
```

Comment:
<img width="825" alt="image" src="https://user-images.githubusercontent.com/64923802/193646304-a2fae675-9018-4053-b60b-df1d3bed5729.png">
I don't think it is the desired result (duplicated commas)